Source/cmake/OptionsCommon.cmake

```
@@ -34,10 +34,20 @@ if (WTF_CPU_ARM)
     endif ()
 endif ()
 
+execute_process(COMMAND uname -m OUTPUT_VARIABLE WTF_HOST_SYSTEM_MACHINE OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+if (("${WTF_HOST_SYSTEM_MACHINE}" STREQUAL "armv8l") OR ("${WTF_HOST_SYSTEM_MACHINE}" STREQUAL "armv7l"))
```

This could be a single `MATCHES` check. Also IIRC the `l` suffix is optional and may change to other things, like `armv7ve`, so I think it's better to match on the prefix only:

```suggestion
if ("${CMAKE_HOST_SYSTEM_PROCESSOR}" MATCHES "^armv[78]")
```

---

We can use `if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "^armv[78]")`

---

Good point, thanks, I'll update the PR.